build.sh

```diff
@@ -81,6 +81,14 @@ for BINARY in $BINARIES; do
     error "Unable to find output"
     exit 1
   fi
+  
+  # MINIFY set to false by default
+  MINIFY=${MINIFY:-"false"}
+  if [ "$MINIFY" = "true" ]; then
+    info "Minifying ${OUTPUT}..."
+    strip "${OUTPUT}"
+    upx "${OUTPUT}"
```

Comment:
Must redirect output to stderr here as stdout is being used to tell entrypoint which files to zip
```suggestion
    strip "${OUTPUT}" >&2
    upx "${OUTPUT}" >&2
```